SPECS/openmpi/openmpi.spec

```diff
@@ -143,10 +129,9 @@ Requires:       python(abi)
 OpenMPI support for Python 3.
 
 %prep
-%autosetup -p1 -n %{name}-%{version}
-%if %{with autogen}
+%autosetup -p1 -n openmpi-%{version}
 ./autogen.pl --force
```

Comment:
Since the original was building without `autogen`, then we should have removed this line along with `%if %{with autogen}`:
```suggestion
```